### ISSUE TITLE
Transfer model for Geothermal Field to AixLib

### ISSUE THREAD

**Issue by AnaConstantin** [RWTH-EBC/AixLib#332]
- [ ] Look into available models, and make a list of models to be transfered

- [ ] Compare the list to available models in Annex60 and analyze if similar modela are available, if models can be combined or substituted through better ones 

- [ ] Transfer the models

**Comment by SebastianRoc** [comment]
Hallo AnaConstantin,

I am working on a model for a thermal ground storage with freezing the included water with Modelica at the moment. That is why I am intersted in your model, because I search for an example in which thermal storage is solved in 3D-model.
Could you provide me the file of the model?
Thanks in advance.

Sebastian

**Comment by AnaConstantin** [comment]
Hello @SebastianRoc . Sorry for the late reply, I was on holiday. I think the Model we want to put into AixLib is close to what you want. Please allow for one more week for the model to be put in the branch.
Afterwards we need to work on it and make sure everything works ok, but you can easily get an idea if it is what you want. If that is the case maybe you can help to develop it further in AixLib.

**Comment by SebastianRoc** [comment]
Hello Ana,
sorry for my late reply, I was very busy to an other project the last weeks.
Could you send me a link of the latest version of model.  I would like to get an idea, whether the programm is to be considered to my problem.
If that is the case I could help to develop it.

**Comment by AnaConstantin** [comment]
Hi Sebastian,

sorry for the late reply. We are behind schedule with the inclusion of the model.
I just came back from my holiday and will start working on it this week.
Please follow issue https://github.com/RWTH-EBC/AixLib/issues/332 for updates.
Best regards
Ana Constantin

Von: SebastianRoc [mailto:notifications@github.com]
Gesendet: Mittwoch, 17. Mai 2017 10:23
An: RWTH-EBC/AixLib <AixLib@noreply.github.com>
Cc: Constantin, Ana <AConstantin@eonerc.rwth-aachen.de>; Author <author@noreply.github.com>
Betreff: Re: [RWTH-EBC/AixLib] Transfer model for Geothermal Field to AixLib (#332)


Hello Ana,
sorry for my late reply, I was very busy to an other project the last weeks.
Could you send me a link of the latest version of model. I would like to get an idea, whether the programm is to be considered to my problem.
If that is the case I could help to develop it.

**Comment by mlauster** [comment]
@AnaConstantin, what is the further plan with this?

**Comment by mlauster** [comment]
@MichaMans, you seem to work on this. Any time schedules when it will be finished? Could you assign yourself?

**Comment by MBaranskiEBC** [comment]
One of our students intends to use and update the old model. @MichaMans, are you planning significant changes?

**Comment by MichaMans** [comment]
@MBaranskiEBC No, don't want to implement some significant changes. The student can test and improve the transferred model and maybe add some docu etc. That would be great

**Comment by DaJansenGit** [comment]
As we might use the model in a current project, I will try to improve the model with the ongoing changes in the branch #751

**Comment by KremerMartin** [comment]
@DaJansenGit Could this issue be something for the hackday next week?

**Comment by DaJansenGit** [comment]
We are currently working on this issue in the B13 project. When the model is finished we will bring it to AixLib but this won't be done next week ;)

**Comment by katharinabr** [comment]
@DaJansenGit What's the status?

**Comment by DaJansenGit** [comment]
development ist still ongoing :)

**Comment by MartinRaetz** [comment]
@DaJansenGit 
Is there a status update?
Should this Issue remain active?

**Comment by DaJansenGit** [comment]
Due to the long simulation times that come with the models and problems in the existing implementation we decided to not use them in our project. I tried to convert them to FastHVAC but didn't completed this as we decided that for our focus we didn't need the detailed invenstigation of borefields.
Currently I have no capacity to continue work on this problem.  
@MichaMans Do you have usage for these models currently?

**Comment by phillipstoffel** [comment]
Maybe I'll take another look at the model in the future, but I can't promise. So add stale.

**Comment by MichaMans** [comment]
@DaJansenGit No, I don't. Stale is fine for me

**Comment by SebastianRoc** [comment]
I am working on a fundamentally different project and unfortunately i no longer need the model.
